bazel/envoy_binary.bzl

```diff
@@ -69,6 +69,7 @@ def _envoy_linkopts():
         "@envoy//bazel:windows_x86_64": [
             "-DEFAULTLIB:ws2_32.lib",
             "-DEFAULTLIB:iphlpapi.lib",
+            "-DEFAULTLIB:shell32.lib",
```

Comment:
just curious what the shell includes are needed for?

Reply:
if it is needed, you might have to add it to the `opt` compile mode section above this one as well:

```
"@envoy//bazel:windows_opt_build": [
            "-DEFAULTLIB:ws2_32.lib",
            "-DEFAULTLIB:iphlpapi.lib",
            "-DEBUG:FULL",
            "-WX",
        ],
```

Reply:
We need it for `CommandLineToArgvW` which is used to parse the argument string. I will have to add it

Reply:
thanks fixed in aba15d9ecf2fb289bfbe0e79ecde24531681c7e6